src/metabase/api/setup.clj

```diff
@@ -162,26 +168,32 @@
              [:dashboard :boolean]
              [:pulse :boolean]
              [:hidden-table :boolean]
-             [:collection :boolean]]]])
+             [:collection :boolean]
+             [:embedded-resource :boolean]]]])
 
 (mu/defn ^:private state-for-checklist :- ChecklistState
   []
-  {:db-type    (mdb/db-type)
-   :hosted?    (premium-features/is-hosted?)
-   :configured {:email (email/email-configured?)
-                :slack (slack/slack-configured?)}
-   :counts     {:user  (t2/count User)
-                :card  (t2/count Card)
-                :table (t2/count Table)}
-   :exists     {:non-sample-db (t2/exists? Database, :is_sample false)
-                :dashboard     (t2/exists? Dashboard)
-                :pulse         (t2/exists? Pulse)
-                :hidden-table  (t2/exists? Table, :visibility_type [:not= nil])
-                :collection    (t2/exists? Collection)
-                :model         (t2/exists? Card :type :model)}})
+  {:db-type                              (mdb/db-type)
+   :hosted?                              (premium-features/is-hosted?)
+   :interested-in-embedding              (not (= (embed.settings/embedding-homepage) :hidden))
```

Comment:
"hidden" is the default value, if the during the setup the user is interested in embedding then the values changes to "visible" and from that it can only go to "dismissed-${reason}" and never back to "hidden"